SQL_SELECT_Mariam_Poghosyan_HW.sql

```diff
@@ -0,0 +1,133 @@
+--1-1
+SELECT f.title

```

Comment:
It is much better to use the entire list of columns that appears in the conditions in the selection. In this case, release_year would be appropriate.